File: Bars.Gkh/Entities/Dicts/OrganizationForm.cs
namespace Bars.Gkh.Entities
{
    /// <summary>
    /// Организационно правовая форма
    /// </summary>
    public class OrganizationForm : BaseGkhEntity
    {
        /// <summary>
        /// Наименование
        /// </summary>
        public virtual string Name { get; set; }

        /// <summary>
        /// Код
        /// </summary>
        public virtual string Code { get; set; }

        /// <summary>
        /// Код ОКОПФ
        /// </summary>
        public virtual string OkopfCode { get; set; }
    }
}

File: Bars.Gkh/libs/B4/model/Dict/OrganizationForm.js
Ext.define('B4.model.dict.OrganizationForm', {
    extend: 'B4.base.Model',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'organizationForm'
    },
    fields: [
        { name: 'Id', useNull: true },
        {
            name: 'Name'
        },
        {
            name: 'Code'
        },
        {
            name: 'OkopfCode'
        }
    ]
});